frontend/src/metabase/forms/components/Form/Form.tsx

```diff
@@ -9,13 +11,22 @@ export interface FormProps
   extends BoxProps,
     FormHTMLAttributes<HTMLFormElement> {
   disabled?: boolean;
+  onValuesChange?: (vals: any) => void;
```

Comment:
is there any way to tighten this up a bit? like

```suggestion
  onValuesChange?: (vals: Record<string, unknown>) => void;
```

?